apps/studio/package.json

```diff
@@ -114,6 +114,7 @@
     "@trpc/server": "10.45.2",
     "@uidotdev/usehooks": "^2.4.1",
     "ajv": "^8.17.1",
+    "csv-parser": "^3.2.0",
```

Comment:
unrelated but im starting to see a benefit of splitting out `scripts` into a standalone project i this monorepo instead of having Studio package.json bloats up ahaha